src/components/FileManager/FileManager.tsx

```diff
@@ -50,17 +50,39 @@ export type Move = {
  * 2- Supplemental files.
  * 3- Other files.
  */
+
+export enum FileSectionIndex {
+  'inline-files' = 0,
+  'supplements-files' = 1,
+  'other-files' = 2,
+}
+
 export const FileManager: React.FC = () => {
-  const [{ doc, files }] = useStore((s) => ({
+  const [{ doc, files, inspectorOpenTabs }] = useStore((s) => ({
     doc: s.doc,
     files: s.files,
+    inspectorOpenTabs: s.inspectorOpenTabs,
   }))
 
   const { figures, supplements, others } = groupFiles(doc, files)
 
+  const initialIndex = 0
+  const [activeTab, setActiveTab] = useState(initialIndex)
+
+  useEffect(() => {
+    if (inspectorOpenTabs?.secondaryTab) {
+      const newIndex =
+        FileSectionIndex[
+          inspectorOpenTabs.secondaryTab as keyof typeof FileSectionIndex
+        ]
+      setActiveTab(newIndex)
+    }
+  }, [inspectorOpenTabs?.secondaryTab])
+
   return (
     <InspectorTabs
       defaultIndex={0}
+      selectedIndex={activeTab}
```

Comment:
if selectedIndex is needed only inside InspectorTabs, why not to produce it right there?